plugins/agentic-docs/skills/component-docs/SKILL.md

```
@@ -63,495 +62,263 @@ component-repo/
-
-**Files**: `[COMPONENT]_DEVELOPMENT.md`, `[COMPONENT]_TESTING.md`
-
-**Templates**: Use `templates/DEVELOPMENT-template.md` and `templates/TESTING-template.md`
-
-### DEVELOPMENT.md Contents
-
-**Required sections** (component-specific ONLY):
+- [ ] **Ask user**: "Run `/review-docs` to verify claims?"
+  - If **YES**: Run `/review-docs --path "$REPO_PATH"`
+  - If **NO**: Warn user:
+    ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)